lib/writer.js

```diff
@@ -4,12 +4,16 @@ var Bluebird = require('bluebird');
 
 var DEFAULT_TYPE = 'other';
 var TYPES = {
+  build: 'Build system / dependencies',
+  ci: 'Continuous Integration',
   chore: 'Chores',
   docs: 'Documentation Changes',
   feat: 'New Features',
   fix: 'Bug Fixes',
   other: 'Other Changes',
+  perf: 'Perfomance improvements',
```

Comment:
Same as above, can you capitalize `Improvements`?

Also, looks like you misspelled `Performance` 😆